backend/connector-integration/src/connectors/peachpayments/transformers.rs

```
@@ -0,0 +1,899 @@
+                    website_url: connector_merchant_config.website_url,
+                };
+
+                // Get routing configuration from metadata
+                let routing = Routing {
+                    route: connector_merchant_config.route,
+                    mid: connector_merchant_config.mid,
+                    tid: connector_merchant_config.tid,
+                    visa_payment_facilitator_id: connector_merchant_config
+                        .visa_payment_facilitator_id,
+                    master_card_payment_facilitator_id: connector_merchant_config
+                        .master_card_payment_facilitator_id,
+                    sub_mid: connector_merchant_config.sub_mid,
+                    amex_id: connector_merchant_config.amex_id,
+                };
+
+                let card = CardDetails {
+                    pan: req_card.card_number.clone(),
+                    cardholder_name: req_card.card_holder_name.clone(),
+                    expiry_year: req_card.card_exp_year.clone(),
+                    expiry_month: req_card.card_exp_month.clone(),
+                    cvv: Some(req_card.card_cvc.clone()),
+                };
+
+                let amount = AmountDetails {
+                    amount: amount_in_cents,
+                    currency_code: item.router_data.request.currency.to_string(),
+                    display_amount: None,
+                };
+
+                let ecommerce_data = EcommerceCardPaymentOnlyTransactionData {
+                    merchant_information,
+                    routing,
+                    card,
+                    amount,
+                };
+
+                // Generate current timestamp for sendDateTime (ISO 8601 format: YYYY-MM-DDTHH:MM:SSZ)
+                let send_date_time = OffsetDateTime::now_utc()
+                    .format(&time::format_description::well_known::Iso8601::DEFAULT)
+                    .map_err(|_| errors::ConnectorError::RequestEncodingFailed)?;
```

Don't discard the errors. Use change context